From; Tzafrir Cohen <tzafrir@debian.org>
Subject: Enable the test framework

Note that enabling this patch breaks the ABI hash. Take care.

Test modules are normally only enabled in development mode. This patch
enables them without the need to enable dev-mode.

The modules provide simple tests. Try: asterisk -rx 'test execute all'

Those modules should not be confused with the tests in the
asterisk-testsute package.

--- a/build_tools/cflags-devmode.xml
+++ b/build_tools/cflags-devmode.xml
@@ -24,7 +24,4 @@
 		<member name="THREAD_CRASH" displayname="Crash on mutex errors">
 			<support_level>extended</support_level>
 		</member>
-		<member name="TEST_FRAMEWORK" displayname="Enable Test Framework API">
-			<support_level>extended</support_level>
-		</member>
 	</category>
--- a/build_tools/cflags.xml
+++ b/build_tools/cflags.xml
@@ -101,4 +101,8 @@
 			<defaultenabled>yes</defaultenabled>
 			<depend>native_arch</depend>
 		</member>
+		<member name="TEST_FRAMEWORK" displayname="Enable Test Framework API">
+			<defaultenabled>yes</defaultenabled>
+			<support_level>extended</support_level>
+		</member>
 	</category>
